mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -18,7 +20,7 @@ CLASS none/ato net/minecraft/world/chunk/Chunk
 		ARG 2 z
 	METHOD <init> (Lnone/aiv;Lnone/auo;II)V
 		ARG 0 world
-	METHOD a matches (II)Z
+	METHOD a matchesPos (II)Z
```

Comment:
I wanted to avoid anyone thinking anything beyond the position is checked to match, which is what it may look like when you first see the method being used (without looking at its code).